src/lambda_function/events/generate_test_event.py

```diff
@@ -0,0 +1,138 @@
+"""
+This is a utility script that generates a fake S3 event notification and writes
+it out as JSON. This test event can be used to test the lambda which
```

Comment:
Can you speak to how the json can be used to test the lambda?

Reply:
I expanded the docstring a bit and left a link to the README (which I also updated).